lms/djangoapps/courseware/access.py

```diff
@@ -173,53 +182,6 @@ def has_access(user, action, obj, course_key=None):
                     .format(type(obj)))
 
 
-# ================ Implementation helpers ================================
-
-def has_staff_access_to_preview_mode(user, obj, course_key=None):
```

Comment:
Need to be more sure about draft version access but I think preview mode may display draft as preview mode is representation of how a course will look after it is published. However a live view should not display draft.